presto-main/src/main/java/com/facebook/presto/execution/GuiceFetcherModule.java

```diff
@@ -0,0 +1,16 @@
+package com.facebook.presto.execution;
+
+import com.facebook.presto.server.remotetask.ContinuousBatchTaskStatusFetcher;
```

Comment:
remove this and add this line in ServerMainModule
`binder.bind(ContinuoustTaskStatusFetcher.class).in(Scopes.SINGLETON);`